src/main/resources/public/jsx/util/form.jsx

```diff
@@ -81,4 +90,127 @@ SubmitButton.propTypes = {
     label: React.PropTypes.string.isRequired
 }
 
-module.exports = { Form, InputText, Select, SubmitButton }
+
+const InputDatePicker = ({ label, labelClassName, divClassName, name, startDate, onChange, dropdownMode }) =>
+    <div className='form-group'>
+        <label className={labelClassName}>
+            {label}
+        </label>
+        <div className={divClassName}>
+            <DatePicker selected={startDate}
+                        dateFormat="DD/MM/YYYY"
+                        onChange={onChange}
+                        dropdownMode={dropdownMode}
+                        className="form-control" name={name} />
+        </div>
+    </div>
+
+InputDatePicker.propTypes = {
+    name: React.PropTypes.string.isRequired,
+    value: React.PropTypes.string,
+    dropdownMode: React.PropTypes.string,
+    label: React.PropTypes.string.isRequired,
+    startDate: React.PropTypes.object,
+    labelClassName: React.PropTypes.string,
+    divClassName: React.PropTypes.string,
+    onChange: React.PropTypes.func
+}
+
+InputDatePicker.defaultProps = {
+    divClassName: 'col-sm-7',
+    labelClassName: 'control-label col-sm-3',
+    dropdownMode: 'scroll'
+}
+
+
+var CountrySelect = React.createClass({
```

Comment:
For new components, use the recommended React way (class XX extends React.Component)